docs/manage/partitions/index.md

```diff
@@ -60,55 +60,55 @@ In this section, we'll introduce the following concepts:
 <div className="box-wrapper" >
```

Comment:
This entire section of cards needs to be re-done to match the contents of the left nav.

Also, all the content above the "## Guides" section should be placed into an "Overview" article.

Reply:
Currently some sections from this doc is moved to different doc. And we will continue to keep the same doc as a landing page for Data tiers for now, because we have the similar set of page for partitions and flex. Also, some existing folders under Manage has similar approach. May be we can clean these things in future in one shot.